vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt

```diff
@@ -154,13 +155,19 @@ class VectorJitsiActivity : VectorBaseActivity<ActivityJitsiBinding>(), JitsiMee
     }
 
     override fun onConferenceTerminated(p0: MutableMap<String, Any>?) {
-        finish()
+        Timber.v("JitsiMeetViewListener.onConferenceTerminated()")
+        // Do not finish if there is an error
+        if (p0?.get("error") == null) {
```

Comment:
The jisti view already show an error?  otherwise, we should maybe show one

Reply:
Yes, it shows an error... But we cannot be a hundred percent sure about that for all cases... So I do not know. This could be improved in the future, there are other useful information than we could display: someone's joined the conf, someone left it, etc.